app/services/withdraw_participant.rb

```diff
@@ -61,6 +62,21 @@ def not_already_withdrawn
     errors.add(:participant_profile, I18n.t(:invalid_withdrawal)) if participant_profile.withdrawn_for?(cpd_lead_provider:)
   end
 
+  def with_started_participant_declarations
+    return unless participant_profile && participant_profile.npq?
+
+    errors.add(:participant_profile, I18n.t(:no_started_declaration_found)) unless any_participant_declarations_started?
+  end
+
+  def any_participant_declarations_started?
+    participant_profile
+      .participant_declarations
+      .where(
+        course_identifier:,
+        declaration_type: "started",
+      ).any?
```

Comment:
Would `#exists?` make sense here?

```suggestion
      ).exists?
```